src/app/portlets/dot-apps/dot-apps-configuration-detail/dot-apps-configuration-detail-form/dot-apps-configuration-detail-form.component.html

```diff
@@ -53,7 +53,9 @@
                     <textarea
                         #inputTextarea
                         pInputTextarea
-                        oninput='this.style.height = "";this.style.height = this.scrollHeight + "px"'
+                        autoResize="autoResize"
```

Comment:
dont we have any test failing for this change?

Reply:
@freddyucv done!